slactwin/db_api.py

```diff
@@ -0,0 +1,25 @@
+"""database service
+
+:copyright: Copyright (c) 2024 The Board of Trustees of the Leland Stanford Junior University, through SLAC National Accelerator Laboratory (subject to receipt of any required approvals from the U.S. Dept. of Energy).  All Rights Reserved.
+:license: http://github.com/slaclab/slactwin/LICENSE
+"""
+
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdlog, pkdp
+import datetime
+import slactwin.quest
+
+
+class DbAPI(slactwin.quest.API):
+    async def api_run_kinds_and_values(self, api_arg):
```

Comment:
I think api_arg needs to be documented.

Reply:
https://github.com/slaclab/slactwin/issues/10